examples/pytorch_lightning/vicregl.py

```diff
@@ -37,8 +37,8 @@ def forward(self, x):
 
     def training_step(self, batch, batch_index):
         (view_global, view_local, grid_global, grid_local), _, _ = batch
-        z_global, z_global_local_features = model(view_global)
-        z_local, z_local_local_features = model(view_local)
+        z_global, z_global_local_features = self(view_global)
```

Comment:
```suggestion
        z_global, z_global_local_features = self.forward(view_global)
        z_local, z_local_local_features = self.forward(view_local)
```
to make it more explicit